server/models/Transaction.ts

```diff
@@ -1006,6 +1039,87 @@ Transaction.createHostFeeTransactions = async (transaction, host, data) => {
   return { transaction, hostFeeTransaction };
 };
 
+/**
+ * Returns the Vendor account associated with a payment processor service. This function is memoized as
+ * the processor fee vendor will not change during the lifetime of the server.
+ */
+Transaction.getPaymentProcessorFeeVendor = memoize(
+  async (service: PAYMENT_METHOD_SERVICE | PayoutMethodTypes | 'OTHER'): Promise<Collective> => {
+    const vendorSlugs = {
+      [PAYMENT_METHOD_SERVICE.STRIPE]: 'stripe-payment-processor-vendor',
+      [PAYMENT_METHOD_SERVICE.PAYPAL]: 'paypal-payment-processor-vendor',
+      [PayoutMethodTypes.BANK_ACCOUNT]: 'wise-payment-processor-vendor', // TODO: We should differentiate depending on manual vs. automatic
```

Comment:
@kewitz Is there a way we could detect whether it's Wise or manual from the transaction `data`, without async code?

Reply:
TODO!